tutorials/hoon/async-monad.md

```diff
@@ -4,7 +4,11 @@ weight = 37
 template = "doc.html"
 +++
 
-Gall apps can be difficult to understand and simple ones often don't need the full complexity available when writing standard Gall apps. To that end, the `tapp` library implements the "async monad" which is more accurately described as a transitional IO monad. If you're not familiar with monads, don't worry as that understanding will not be necessary to use the `tapp` library.
+Note: as of 2019.12.02, this material is out of date. While this is being updated, please see the [Gall overview](@/docs/tutorials/arvo/gall.md) in the Arvo section to get oriented with Gall. Some material presented here may still be useful.
```

Comment:
Yes, the async monad is superseded by "threads" in static gall, as mentioned in #737.  It has the same basic mechanics but significant changes.  The biggest changes are: instead of being a core with `+handle-poke` and the other arms, it's just a function; threads don't last forever, they return a value; "tapp" -> "thread"; "async" -> "strand".